arviz/stats/stats.py

```diff
@@ -61,19 +61,20 @@ def compare(
     scale: Optional[ScaleKeyword] = None,
     var_name: Optional[str] = None,
 ):
-    r"""Compare models based on PSIS-LOO `loo` or WAIC `waic` cross-validation.
+    r"""Compare models based on  their expected log pointwise predictive density (ELPD).
 
-    LOO is leave-one-out (PSIS-LOO `loo`) cross-validation and
-    WAIC is the widely applicable information criterion.
-    Read more theory here - in a paper by some of the leading authorities
-    on model selection dx.doi.org/10.1111/1467-9868.00353
+
+    The ELPD is estimated either by Pareto smoothed importance sampling leave-one-out
+    cross-validation (loo) or using the widely applicable information criterion (waic).
```

Comment:
cross-validation (~loo~LOO) or using the widely applicable information criterion (~waic~WAIC).